trl/trainer/dpo_trainer.py

```diff
@@ -100,12 +107,17 @@ class DPOTrainer(Trainer):
         compute_metrics (`Callable[[EvalPrediction], Dict]`, *optional*):
             The function to use to compute the metrics. Must take a `EvalPrediction` and return
             a dictionary string to metric values.
+        model_kwargs: (`Optional[Dict]`, *optional*):
```

Comment:
I'd be in favour of calling it `model_init_kwargs` to make it clear where they are used.

Reply:
done